python/mass_nifti_pic.py

```diff
@@ -159,17 +166,23 @@ def make_pic(file_id, config_file, verbose):
     # load the Imaging object
     imaging = Imaging(db, verbose)
 
+    # checks if there is already a pic for the NIfTI file
+    existing_pic_file_in_db = imaging.grep_parameter_value_from_file_id(file_id, 'check_pic_filename')
+
     # grep the NIfTI file path
     nii_file_path = imaging.grep_file_path_from_file_id(file_id)
     if not nii_file_path:
-        print('WARNING: no file in the database with FileID = ' + file_id)
+        print('WARNING: no file in the database with FileID = ' + str(file_id))
         return
     if not re.search('.nii.gz$', nii_file_path):
         print('WARNING: wrong file type. File ' + nii_file_path + ' is not a .nii.gz file')
         return
     if not os.path.exists(data_dir + nii_file_path):
         print('WARNING: file ' + nii_file_path + ' not found on the filesystem')
         return
+    if existing_pic_file_in_db and not force:
```

Comment:
I would move the declaration of `existing_pic_file_in_db` closer to the line where it isa first used (i.e. here). Just nitpicking...  :)

Reply:
@nicolasbrossard done in the last commit! Thank you! :)